src/server/rest/v1/service/UtilsService.ts

```diff
@@ -101,6 +101,82 @@ export default class UtilsService {
     return chargingStation;
   }
 
+  public static async checkAndGetChargingStationsAuthorization(tenant: Tenant, userToken: UserToken, action: ServerAction,
+      additionalFilters: Record<string, any>, applyProjectFields = false): Promise<ChargingStation[]> {
+    // Check dynamic auth
+    const authorizationFilter = await AuthorizationService.checkAndGetChargingStationsAuthorizationFilters(tenant, userToken, {});
+    if (!authorizationFilter.authorized) {
+      throw new AppAuthError({
+        errorCode: HTTPAuthError.FORBIDDEN,
+        user: userToken,
+        action: Action.LIST,
+        entity: Entity.CHARGING_STATIONS,
+        module: MODULE_NAME, method: 'checkAndGetChargingStationsAuthorization',
+      });
+    }
+    // Get Charging Stations
+    const chargingStations = await ChargingStationStorage.getChargingStations(tenant.id,
+      {
+        ...additionalFilters,
+        ...authorizationFilter.filters,
+      }, Constants.DB_PARAMS_MAX_LIMIT,
+      applyProjectFields ? authorizationFilter.projectFields : null
+    );
+    // Check
+    for (const chargingStation of chargingStations.result) {
```

Comment:
This method will be used to get the ChargingStations from ChargingStationService endpoint and can return the ChargingStations with issuer = true or false.
This check has to be done in method checkSiteAreaChargingStationsAuthorization.
You cannot assign Charging Stations to Site Area which have issuer flag set to false but you can list them in Charging Station list (Owner filter)